apps/meteor/client/views/omnichannel/currentChats/CurrentChatsRoute.tsx

```diff
@@ -182,36 +168,109 @@ const CurrentChatsRoute = (): ReactElement => {
 			};
 
 			return (
-				<Table.Row key={_id} tabIndex={0} role='link' onClick={(): void => onRowClick(_id)} action qa-user-id={_id}>
-					<Table.Cell withTruncatedText>{fname}</Table.Cell>
-					<Table.Cell withTruncatedText>{department ? department.name : ''}</Table.Cell>
-					<Table.Cell withTruncatedText>{servedBy?.username}</Table.Cell>
-					<Table.Cell withTruncatedText>{moment(ts).format('L LTS')}</Table.Cell>
-					<Table.Cell withTruncatedText>{moment(lm).format('L LTS')}</Table.Cell>
-					<Table.Cell withTruncatedText>{getStatusText(open, onHold)}</Table.Cell>
-					{canRemoveClosedChats && !open && <RemoveChatButton _id={_id} reload={reload} />}
-				</Table.Row>
+				<GenericTableRow key={_id} onClick={(): void => onRowClick(_id)} action>
+					<GenericTableCell withTruncatedText>{fname}</GenericTableCell>
+					<GenericTableCell withTruncatedText>{department ? department.name : ''}</GenericTableCell>
+					<GenericTableCell withTruncatedText>{servedBy?.username}</GenericTableCell>
+					<GenericTableCell withTruncatedText>{moment(ts).format('L LTS')}</GenericTableCell>
+					<GenericTableCell withTruncatedText>{moment(lm).format('L LTS')}</GenericTableCell>
+					<GenericTableCell withTruncatedText>{getStatusText(open, onHold)}</GenericTableCell>
+					{canRemoveClosedChats && !open && (
+						<GenericTableCell withTruncatedText>
+							<RemoveChatButton _id={_id} reload={reload} />
+						</GenericTableCell>
+					)}
+				</GenericTableRow>
 			);
 		},
-		[onRowClick, reload, canRemoveClosedChats, t],
+		[canRemoveClosedChats, onRowClick, reload, t],
 	);
 
 	if (!canViewCurrentChats) {
 		return <NotAuthorizedPage />;
 	}
 
-	return id ? (
+	return id && id !== 'custom-fields' ? (
```

Comment:
Just ponting out that`id !== 'custom-fields'` already covers both conditions.

Reply:
Not exactly as `undefined !== 'custom-fields'` and `id` should not be undefined